lib/DDG/Meta/Data.pm

```diff
@@ -161,6 +161,13 @@ sub get_ia {
     return clone($m);
 }
 
+sub get_js {
+    my ($self, $id) = @_;
+
+    my $metaj = encode_json($self->get_ia(id => $id));                                                                                                                                                                                              
```

Comment:
As one aside, this can cause a `die` which could kill any process running this that doesn't `eval`...